cmplibrary/src/main/java/com/sourcepoint/cmplibrary/model/exposed/SpConfig.kt

```diff
@@ -17,8 +17,14 @@ data class SpConfig(
 
 data class SpCampaign(
     @JvmField val campaignType: CampaignType,
-    @JvmField internal val targetingParams: List<TargetingParam>
-)
+    @JvmField internal val targetingParams: List<TargetingParam>,
+    @JvmField val groupPmId: Long? = null
+) {
+    constructor(
+        campaignType: CampaignType,
+        targetingParams: List<TargetingParam>
+    ) : this(campaignType, targetingParams, null)
```

Comment:
Should `groupPmId` be part of the `constructor`?

Reply:
The `groupId` tells us that the property has a group of campaigns.
This value should be bound to the property rather to the campaigns

Reply:
Right, that makes sense. The group pm id is bound to property and the child pm id is a campaign attribute (which the backend return in the UserConsent).